.gitignore

```diff
@@ -18,3 +18,5 @@ coraza-waf
 __debug_bin
 
 build/
+
+go.work.sum
```

Comment:
Shall we?

Reply:
I'm facing quite often diffs compared to master and as far as I understood it is kind of meant to be just local. But I can just revert this and push changes

Reply:
Sure let's remove the file then

Reply:
oops, done!